### PR TITLE
[JA] Translate ecto/changesets

### PR DESCRIPTION
**Pull request by Tsuyoshi84**
I translated `ecto/changesets` section!
Please review! @elixirschool/japanese

### CODE REVIEW

File: ja/lessons/ecto/changesets.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+`User` スキーマを使うチェンジセットを作るためには、 `Ecto.Changeset.cast/4` を使います:
+
+```elixir
+iex> Ecto.Changeset.cast(%User{name: "Bob"}, %{}, [:name])
+#Ecto.Changeset<action: nil, changes: %{}, errors: [], data: #User<>,
+ valid?: true>
+```
+
+最初のパラメータは元のデータで、この場合は空の `%User{}` 構造体です。
+Ectoには構造体そのものに基づいてスキーマを見つける賢さがあります。
```
[the remark this answers is not in the document]
Comment (Tsuyoshi84) [reply ja/lessons/ecto/changesets.md]: I agree with you! I'll fix it!

File: ja/lessons/ecto/changesets.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+```elixir
+def changeset(struct, params) do
+  struct
+  |> cast(params, [:name])
+  |> validate_required([:name])
+  |> validate_length(:name, min: 2)
+end
+```
+
+nameに対して1つの文字を渡した場合、どのような結果になるかを試して推測することができます！
```
[the remark this answers is not in the document]
Comment (Tsuyoshi84) [reply ja/lessons/ecto/changesets.md]: That sounds better 👍

File: ja/lessons/ecto/changesets.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+  data: #User<>,
+  valid?: false
+>
+```
+
+エラーメッセージが暗号のような `%{count}` を含んでいることに驚くかもしれません。これは他言語への翻訳を補助するためです。ユーザーに直接エラーを表示したい場合、 [`traverse_errors/2`](https://hexdocs.pm/ecto/Ecto.Changeset.html#traverse_errors/2) を使って人が読める形式に変更できます。ドキュメントで提供されている例に目を通してください。
+
+`Ecto.Changeset` にある他のビルトインのバリデーションは、以下のものがあります:
+
+- validate_acceptance/3
```
[the remark this answers is not in the document]
Comment (Tsuyoshi84) [reply ja/lessons/ecto/changesets.md]: Oh, I'm not sure why I did this but this is not my intention.
I'll fix it!

**Comment by Tsuyoshi84** [comment]
I fixed the translation!
Thank you for your super helpful review as always!

**Comment by riseshia** [comment]
IMO "アソシエーション" 👍

**Comment by Tsuyoshi84** [comment]
Thanks, I got it!
I fixed the line 289 :)

**Comment by riseshia** [comment]
Thanks!!!

**Comment by doomspork** [comment]
Thank you so much @Tsuyoshi84 and @riseshia — you're the best!

**Pull request merged**